tests/src/actionContainers/SwiftActionContainerTests.scala

```diff
@@ -30,9 +30,9 @@ class SwiftActionContainerTests extends BasicActionRunnerTests with WskActorSyst
 
     // note: "out" will likely not be empty in some swift build as the compiler
     // prints status messages and there doesn't seem to be a way to quiet them
-    val enforceEmptyOutputStream = true
-    lazy val swiftContainerImageName = "swiftaction"
-    lazy val envCode = makeEnvCode("NSProcessInfo.processInfo()")
+    val enforceEmptyOutputStream = false
+    lazy val swiftContainerImageName = "swift3action"
+    lazy val envCode = makeEnvCode("ProcessInfo.processInfo")
```

Comment:
Now that we only have Swift 3, we may not need this indirection anymore.  I vote to keep it since we may need stuff like this for Swift 4.